src/hotspot/share/nmt/virtualMemoryTracker.cpp

```diff
@@ -338,6 +337,8 @@ bool VirtualMemoryTracker::add_reserved_region(address base_addr, size_t size,
   assert(base_addr != nullptr, "Invalid address");
   assert(size > 0, "Invalid size");
   assert(_reserved_regions != nullptr, "Sanity check");
+  assert(!MemTracker::is_done_bootstrap() || NmtVirtualMemory_lock->owned_by_self() , "Should have acquired NmtVirtualMemory_lock");
+
```

Comment:
This line is kind of long.  And why the space before the comma?  And there's a bunch of
these, suggesting there should be a helper to package this up.

Reply:
Good point. I've gotten rid of this in favor of the helper that Coleen suggested below.